domain/tests/src/Unit/Access/DomainAccessCheckTest.php

```diff
@@ -0,0 +1,148 @@
+<?php
```

Comment:
OK, now this is just overreaching within a single Pull Request.

This test addition -- though desirable -- has nothing to do with the rest of the PR and needs to be separated into it's own PR.